src/mlpack/methods/adaboost/adaboost_impl.hpp

```diff
@@ -256,14 +256,12 @@ void AdaBoost<WeakLearnerType, MatType>::Classify(
       probabilities(tempPredictedLabels(j), j) += alpha[i];
   }
 
-  arma::colvec pRow;
   arma::uword maxIndex = 0;
 
   for (size_t i = 0; i < predictedLabels.n_cols; ++i)
   {
     probabilities.col(i) /= arma::accu(probabilities.col(i));
-    pRow = probabilities.unsafe_col(i);
-    pRow.max(maxIndex);
+    probabilities.unsafe_col(i).max(maxIndex);
```

Comment:
@rcurtin It might be more efficient to simply use `probabilities.col(i)` instead of `probabilities.unsafe_col(i)`.  This will avoid generation of the fake column vector, and in turn avoid potential performance issues with the move constructor in older versions of Armadillo.

Armadillo subviews have the `.max()` member functions for a long while.  The functions are in version 8.200, and probably in much earlier versions as well.

Reply:
Ah, I thought I did that!  I guess I forgot.  Done in 7db8928.  I think before this PR the last time I checked to see if this version of `.max()` existed was I guess in 2013 or something... :smile: